[2024-07-31T09:17:16.773+0900] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-31T09:17:16.785+0900] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_bash.err.report scheduled__2024-07-30T00:00:00+00:00 [queued]>
[2024-07-31T09:17:16.788+0900] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_bash.err.report scheduled__2024-07-30T00:00:00+00:00 [queued]>
[2024-07-31T09:17:16.788+0900] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-31T09:17:16.794+0900] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): err.report> on 2024-07-30 00:00:00+00:00
[2024-07-31T09:17:16.797+0900] {standard_task_runner.py:64} INFO - Started process 3179 to run task
[2024-07-31T09:17:16.800+0900] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'simple_bash', 'err.report', 'scheduled__2024-07-30T00:00:00+00:00', '--job-id', '672', '--raw', '--subdir', 'DAGS_FOLDER/simple_bash.py', '--cfg-path', '/var/folders/qv/0knwm5h50ml0skpppwpbks_r0000gp/T/tmpn5k1sa29']
[2024-07-31T09:17:16.801+0900] {standard_task_runner.py:91} INFO - Job 672: Subtask err.report
[2024-07-31T09:17:16.817+0900] {task_command.py:426} INFO - Running <TaskInstance: simple_bash.err.report scheduled__2024-07-30T00:00:00+00:00 [running]> on host jeongui-MacBookPro.local
[2024-07-31T09:17:16.839+0900] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='simple_bash' AIRFLOW_CTX_TASK_ID='err.report' AIRFLOW_CTX_EXECUTION_DATE='2024-07-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-30T00:00:00+00:00'
[2024-07-31T09:17:16.839+0900] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-31T09:17:16.846+0900] {subprocess.py:63} INFO - Tmp dir root location: /var/folders/qv/0knwm5h50ml0skpppwpbks_r0000gp/T
[2024-07-31T09:17:16.846+0900] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n            echo "err report"\n        ']
[2024-07-31T09:17:16.849+0900] {subprocess.py:86} INFO - Output:
[2024-07-31T09:17:16.851+0900] {subprocess.py:93} INFO - err report
[2024-07-31T09:17:16.851+0900] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-31T09:17:16.852+0900] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-31T09:17:16.858+0900] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=simple_bash, task_id=err.report, run_id=scheduled__2024-07-30T00:00:00+00:00, execution_date=20240730T000000, start_date=20240731T001716, end_date=20240731T001716
[2024-07-31T09:17:16.907+0900] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-07-31T09:17:16.915+0900] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-31T09:17:16.915+0900] {local_task_job_runner.py:222} INFO - ::endgroup::
